cmake/Modules/Packages/KIM.cmake

```diff
@@ -51,6 +51,9 @@ if(DOWNLOAD_KIM)
     INTERFACE_INCLUDE_DIRECTORIES "${INSTALL_DIR}/include/kim-api")
   target_link_libraries(lammps PRIVATE LAMMPS::KIM)
   add_dependencies(LAMMPS::KIM kim_build)
+  if(BUILD_LIB AND NOT BUILD_SHARED_LIBS)
```

Comment:
This `BUILD_LIB` needs to go.

Reply:
hmmm... that statement must have been added through merging your cmake branch after I had checked the cmake files. should be corrected now.